packages/server-wallet/src/wallet/index.ts

```diff
@@ -259,14 +264,50 @@ export class Wallet extends EventEmitter<WalletEvent>
     return await this.store.getOrCreateSigningAddress();
   }
 
+  async createLedgerChannel(
+    args: Pick<CreateChannelParams, 'participants' | 'allocations'>,
+    fundingStrategy: 'Direct' | 'Unfunded' = 'Direct'
+  ): Promise<SingleChannelOutput> {
+    const {participants, allocations} = args;
+
+    const channelNonce = await this.store.nextNonce(participants.map(p => p.signingAddress));
```

Comment:
Should the store methods called in this function be bound in a single transaction? If multiple `createLedgerChannel` calls hit the same DB around the same time, might there be nonce issues?

Reply:
I think the implementation of `nextNonce` implicitly runs in a transaction-like way? Not 100% sure... 

https://github.com/statechannels/statechannels/blob/02a9216dd1061326ed36bd70089830a313ef5eb6/packages/server-wallet/src/models/nonce.ts#L42-L44

Reply:
Postgres will take care of it!